buildpack/start.py

```diff
@@ -159,12 +150,18 @@ def _register_signal_handlers():
             jmx_config_files=databroker_jmx_config_files,
         )
         nginx.configure(m2ee)
+        databroker.update_config(m2ee)
+        databroker.business_events.update_config(
+            m2ee, util.get_vcap_services_data()
+        )
 
         # Start components and runtime
         telegraf.run()
         datadog.run(model_version, runtime_version)
         metering.run()
-        runtime.run(m2ee)
+        logs.run(m2ee)
+        runtime.run(m2ee, logs.get_loglevels())
+        metrics.run(m2ee)
```

Comment:
Actually, metrics ran after the runtime started: https://github.com/mendix/cf-mendix-buildpack/blob/c74eaf9ded2fbd120e529b9bf51e4602a3521fd2/buildpack/runtime.py#L619. Running metrics before the runtime also causes issues in termination tests.

Logs also ran after, but that caused issues since it has to redirect the logs to a FIFO buffer. I've refactored this so that they both run correctly.